crates/red_knot_python_semantic/src/types.rs

```diff
@@ -899,7 +899,7 @@ impl<'db> Type<'db> {
                 Type::SubclassOf(_),
             ) => true,
             (Type::SubclassOf(_), _) | (_, Type::SubclassOf(_)) => {
-                // TODO: Once we have support for final classes, we can determine disjointness in some cases
+                // TODO: Once we have support for final classes, we can determine disjointedness in some cases
```

Comment:
same comment as last time :-)

https://github.com/astral-sh/ruff/pull/14863#discussion_r1875617796

Reply:
There's no such word in the [OED](https://www.oed.com/search/dictionary/?q=disjointness). But then I guess the word is specifically used in a mathematical context.

Reply:
"disjointness" still sounds slightly better to me in this context. but whatever, I think the meaning is perfectly clear with either, and hopefully in the long run we'll just do the TODO anyway :-)

Reply:
I think "disjointedness" is a totally different word with a different meaning, and wouldn't have merged the PR with this change in it. But yeah, let's just do the TODO and get rid of the comment :)

Reply:
Indeed, `disjointness` appears to be used specifically in mathematics for disjoint sets.

Reply:
Reverted in #14906.